src/util/std_types.h

```diff
@@ -767,6 +767,8 @@ class code_typet:public typet
   class parametert;
   typedef std::vector<parametert> parameterst;
 
+  typedef std::vector<irept> exceptionst;
```

Comment:
Shouldn't that be in `java_method_typet`?

Reply:
It is. It's just done in a later commit. If you view all of the file, there are no changes to std_types in this PR.

Reply:
Not a fan of typedefs like this one. It's not much shorter than a fully qualified vector and it gives you less information when it's explicitly specified,

Reply:
Then these changes should be squashed. Adding code and fixing it up within the same PR is rather confusing.